package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "AWS_PROFILE=reactdc node server.js",
+	 "dev": "./node_modules/.bin/webpack --watch -d --progress",
```

Comment:
Stick to 2 spaces in package.json. When it gets regenerated it'll rewrite this line.